packages/core/src/integrations/requestdata.ts

```diff
@@ -38,12 +39,12 @@ const _requestDataIntegration = ((options: RequestDataIntegrationOptions = {}) =
 
   return {
     name: INTEGRATION_NAME,
-    processEvent(event) {
+    processEvent(event, _hint, client) {
       const { sdkProcessingMetadata = {} } = event;
       const { normalizedRequest, ipAddress } = sdkProcessingMetadata;
 
       if (normalizedRequest) {
-        addNormalizedRequestDataToEvent(event, normalizedRequest, { ipAddress }, include);
+        addNormalizedRequestDataToEvent(event, normalizedRequest, { ipAddress }, include, client);
```

Comment:
l: This is also fine, but what if we mutated `include` here instead? IMHO the `addNormalizedRequestDataToEvent` does not need to know of the client etc, we could do:

```js
addNormalizedRequestDataToEvent(event, normalizedRequest, { ipAddress }, { ip: client.getOptions().sendDefaultPii, ...include });
```

or something like this? (not quite, as ip is set by default, but logic wise something like this...) 

But I'm also OK with leaving it as it is here!